clients/src/main/java/org/apache/kafka/clients/consumer/internals/AbstractCoordinator.java

```diff
@@ -713,20 +717,20 @@ public void handle(HeartbeatResponse heartbeatResponse, RequestFuture<Void> futu
                 future.complete(null);
             } else if (error == Errors.GROUP_COORDINATOR_NOT_AVAILABLE
                     || error == Errors.NOT_COORDINATOR_FOR_GROUP) {
-                log.debug("Attempt to heart beat failed for group {} since coordinator {} is either not started or not valid.",
+                log.debug("Attempt to heartbeat failed for group {} since coordinator {} is either not started or not valid.",
```

Comment:
I used capitalized camel-case when describing the request or response (e.g. "JoinGroup request", "Heartbeat response"), and lower case when the word was part of a sentence (e.g. "successfully joined the group", "Attempt to heartbeat failed"). Does that seem reasonable?